From 1b133f2ca15f0a15b05407b2c04521d7de88dfa2 Mon Sep 17 00:00:00 2001
From: Evgeny Vereshchagin <evvers@ya.ru>
Date: Fri, 9 Nov 2018 03:14:04 +0100
Subject: [PATCH] udev-test: remove a check for whether the test is run in a
 container

It's too broad a check that prevents the test from running on Travis CI.

(cherry picked from commit 881886ef08d50951159633248b0f73977c5d6924)

Related: #1642728
---
 test/udev-test.pl | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/test/udev-test.pl b/test/udev-test.pl
index 61bd3d703a..05b3e17188 100755
--- a/test/udev-test.pl
+++ b/test/udev-test.pl
@@ -1646,13 +1646,6 @@ if ($? >> 8 == 0) {
         exit($EXIT_TEST_SKIP);
 }
 
-# skip the test when running in a container
-system("systemd-detect-virt", "-c", "-q");
-if ($? >> 8 == 0) {
-        print "Running in a container, skipping the test.\n";
-        exit($EXIT_TEST_SKIP);
-}
-
 udev_setup();
 
 my $test_num = 1;
